Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM alpine:latest
-
-ENV ISTIO_VERSION 1.9.8
-
-RUN apk update && apk add curl bash coreutils jq
-
-# Get Istio Charts
+FROM nginx:latest
+ENV ISTIO_VERSION 1.10.4
+USER root
```

Comment:
Can this be removed? I think root is the default user that is used.

Reply:
removed and only change to nginx at end of file